src/cplscheme/tests/serial-explicit-coupling-datainit.xml

```diff
@@ -1,21 +1,28 @@
-<?xml version="1.0"?>
-
+<?xml version="1.0" encoding="UTF-8" ?>
 <configuration>
-   <data:scalar name="Data0"/>
-   <data:vector name="Data1"/>
-   <data:scalar name="Data2"/>
-   <mesh name="Mesh">
-      <use-data name="Data0"/>
-      <use-data name="Data1"/>
-      <use-data name="Data2"/>
-   </mesh>
-   <m2n:sockets from="Participant0" to="Participant1"/>
-   <coupling-scheme:serial-explicit>
-      <participants first="Participant0" second="Participant1"/>
-      <time-window-size value="0.1" method="fixed"/>
-      <max-time-windows value="1"/>
-      <exchange data="Data0" mesh="Mesh" from="Participant1" to="Participant0"/>
-      <exchange data="Data1" mesh="Mesh" from="Participant1" to="Participant0" initialize="on"/>
-      <exchange data="Data2" mesh="Mesh" from="Participant0" to="Participant1"/>
-   </coupling-scheme:serial-explicit>
+  <data:scalar name="Data0" />
+
+  <data:vector name="Data1" />
+
+  <data:scalar name="Data2" />
+
+  <mesh name="Mesh">
+    <use-data name="Data0" />
+    <use-data name="Data1" />
+    <use-data name="Data2" />
+  </mesh>
+
+  <m2n:sockets from="Participant0" to="Participant1" />
+
+  <coupling-scheme:serial-explicit>
+    <participants first="Participant0" second="Participant1" />
+
+    <time-window-size value="0.1" method="fixed" />
+
+    <max-time-windows value="1" />
+
+    <exchange data="Data0" mesh="Mesh" from="Participant1" to="Participant0" />
+    <exchange data="Data1" mesh="Mesh" from="Participant1" to="Participant0" initialize="on" />
+    <exchange data="Data2" mesh="Mesh" from="Participant0" to="Participant1" />
+  </coupling-scheme:serial-explicit>
```

Comment:
Could we avoid empty lines within a coupling-scheme tag?

Reply:
This looks like a bug. Grouping should be disabled after the second level.

Reply:
I looked into this.
This mentioned configuration is only a partial configuration as it does not contain the `<precice-configuration>` and `<solver-interface>` tags.
We could add an exception, however this would add more complexity to the formatter.

I would suggest to ignore formatting for these edge-cases.